blocks/to-do/add-and-delete-list-items.js

```diff
@@ -10,6 +10,12 @@ export default function addListItem(inputValue, list) {
 
   // Set attribute
   listItem.classList.add('to-do__list-item');
+  // Default the "data-completed" attribute to false if it's undefined
+  if (completed) {
+    listItem.setAttribute('data-completed', completed);
+  } else {
+    listItem.setAttribute('data-completed', 'false');
+  }
```

Comment:
you're adding the `data-completed` attribute in both cases. so why have the `if/else`? Instead, you can get away with a one-liner

`listItem.setAttribute('data-completed', completed)` or something similar. I'm not sure what `completed` looks like.